packaging/resource_suite_s3_nocache.py

```diff
@@ -1915,25 +1915,26 @@ def test_istream_append__issue_2153(self):
         filename = 'test_istream_append__issue_2153.txt'
         logical_path = os.path.join(self.user0.session_collection, filename)
         content = 'this is the original file for istream'
-        content_append1 = 'line1 appended to the file'
-        content_append2 = 'line2 appended to the file'
+        content_append1 = '\nline1 appended to the file'
+        content_append2 = '\nline2 appended to the file'
 
         try:
             # istream to a new data object and ensure that it is created successfully.
             self.user0.assert_icommand(['istream', 'write', logical_path], input=content)
             self.assertEqual(str(1), lib.get_replica_status(self.user0, filename, replica_number_in_s3))
 
             # append to the file
-            self.user0.assert_icommand(['istream', 'write', '-a', logical_path], input=f'\n{content_append1}')
+            self.user0.assert_icommand(['istream', 'write', '-a', logical_path], input=f'{content_append1}')
 
             # append to the file again
-            self.user0.assert_icommand(['istream', 'write', '-a', logical_path], input=f'\n{content_append2}')
+            self.user0.assert_icommand(['istream', 'write', '-a', logical_path], input=f'{content_append2}')
 
             # verify the file size in ils -l - added two newlines above thus the +2 below
-            self.user0.assert_icommand(['ils', '-l', logical_path], 'STDOUT', str(len(content) + len(content_append1) + len(content_append2)+ 2)) 
+            self.user0.assert_icommand(['ils', '-l', logical_path], 'STDOUT', str(len(content) + len(content_append1) + len(content_append2))) 
 
             # Ensure that the replica actually contains the contents streamed into it.
-            self.user0.assert_icommand(['iget', logical_path, '-'], 'STDOUT_MULTILINE', [content, content_append1, content_append2])
+            self.user0.assert_icommand(['istream', 'read', logical_path], 'STDOUT', [content, content_append1, content_append2])
+            #self.user0.assert_icommand(['iget', logical_path, '-'], 'STDOUT_MULTILINE', [content, content_append1, content_append2])
 
         finally:
```

Comment:
I think we agreed to put an `ils -L` debug or something at the start of the finally block?